modules/time.js

```diff
@@ -0,0 +1,3 @@
+import { DateTime } from './luxon.js';
+
+export default DateTime;
```

Comment:
- [ ] **OPTIONAL** The `import { DateTime } from './luxon.js';` line, can be used directly in the index.js file, then there is no need for time.js file.